.ci/scripts/functional-test.sh

```diff
@@ -25,13 +25,14 @@ STACK_VERSION=${3:-"${BASE_VERSION}"}
 BEAT_VERSION=${4:-"${BASE_VERSION}"}
 ELASTIC_AGENT_VERSION=${5:-"${BASE_VERSION}"}
 GOARCH=${GOARCH:-"amd64"}
+REPORT_PREFIX="${SUITE}_${GOARCH}_${TAGS}"
```

Comment:
What do you think to allow overriding the value only if REPORT_PREFIX is not set? IIUC, 

https://github.com/elastic/e2e-testing/pull/2273/files#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR564 uses `.ci/scripts/functional-test.sh` with only one parameter, so `REPORT_PREFIX="${SUITE}_${GOARCH}_${TAGS}"` = `REPORT_PREFIX=${tags}_ amd64_" but `REPORT_PREFIX` is already in place as it's created on the fly in the jenkinsfile

Reply:
That could be fine too. My only concern is that the script that is run on CI is generated on the fly in the Ansible code, and this is different to the committed script. I'd try to align them as much as possible